.env.sample

```diff
@@ -32,18 +32,16 @@ APIM_MDM_VALUE=
 APIM_MDM_MAX_REDIRECTS=
 APIM_MDM_TIMEOUT=
 
-
-# MDM
-MDM_BASE_URL=
-MDM_API_KEY_HEADER_NAME=
-MDM_API_KEY_HEADER_VALUE=
-MDM_MAX_REDIRECTS=
-MDM_TIMEOUT=
-
-
 # STORAGE
 DTFS_STORAGE_ACCOUNT_NAME=
 DTFS_STORAGE_ACCOUNT_KEY=
 DTFS_STORAGE_FILE_SHARE_NAME=
 DTFS_STORAGE_EXAMPLE_FILE_NAME=
 DTFS_STORAGE_EXAMPLE_FILE_LOCATION_PATH=
+
+# CUSTODIAN
+CUSTODIAN_BASE_URL=
+CUSTODIAN_API_KEY_HEADER_NAME=
+CUSTODIAN_API_KEY_HEADER_VALUE=
+CUSTODIAN_MAX_REDIRECTS=
+CUSTODIAN_TIMEOUT=
```

Comment:
@abhi-markan when you get a chance, can you please set up these variable in GitHub? I can tell you what the values need to be